go.mod

```diff
@@ -20,7 +20,7 @@ require (
 	github.com/stoewer/go-strcase v1.0.2
 	github.com/terraform-providers/terraform-provider-random v0.0.0-20190925200408-30dac3233094
 	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
-	google.golang.org/api v0.11.1-0.20191012001332-5e0e3f4a3bb8
+	google.golang.org/api v0.13.0
```

Comment:
This was bumped already in TPGB, so it doesn't get changed there. The new BT rev bumped it here, I didn't do it manually.